apps/web/src/actions/rewards/updateRewardClaimValidityAction.ts

```diff
@@ -0,0 +1,28 @@
+'use server'
+
+import { ClaimedReward } from '@latitude-data/core/browser'
+import { UnauthorizedError } from '@latitude-data/core/lib/errors'
+import { updateRewardClaim } from '@latitude-data/core/services/claimedRewards/update'
+import { z } from 'zod'
+
+import { authProcedure } from '../procedures'
+
+export const updateRewardClaimValidityAction = authProcedure
+  .createServerAction()
+  .input(
+    z.object({
+      claimId: z.number(),
+      isValid: z.boolean().nullable(),
+    }),
+  )
+  .handler(async ({ input, ctx }) => {
+    if (!ctx.user.admin) {
```

Comment:
Ok the future is now

Reply:
Also move backoffice actions under `actions/backoffice` to have a more clear difference